Library/Homebrew/cmd/install.rb

```diff
@@ -98,10 +98,36 @@ def install
       else
         ofail e.message
         query = query_regexp(e.name)
-        ohai "Searching formulae..."
-        puts_columns(search_formulae(query))
+
+        ohai "Searching for similarly named formulae..."
+        formulae_search_results = search_formulae(query)
+        case formulae_search_results.length
+        when 0
+          ofail "No similarly named formulae found."
+        when 1
+          puts "This similarly named formula was found:"
+          puts_columns(formulae_search_results)
+          puts "If you meant to install it, run `brew install #{formulae_search_results.first}`"
+        else
+          puts "These similarly named formulae were found:"
+          puts_columns(formulae_search_results)
+          puts "If you meant to install one of them, run (for example) `brew install #{formulae_search_results.first}`"
```

Comment:
Might be worth checking this still looks nice when wrapped at 80 columns (default terminal size) and otherwise add newlines. Could also use `e.g.` instead of `(for example)`